charts/spire/charts/spiffe-oidc-discovery-provider/templates/test-config.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
Does it work if this is moved under tests/ ?

Would be a little bit cleaner that way

Reply:
I think I linked before, only pods (and jobs?) are created when nested under tests subdirectory. It's possible to embed this in the pod, not sure it's worth the impact on readability though.

Reply:
ah. ok. I don't think I saw that. Sorry.

No objections then.